R/compute_centiloid.R

```diff
@@ -0,0 +1,176 @@
+#' Compute Centiloid Value
+#'
+#' Computes centiloid values based on amyloid PET tracer, SUVR value and pipeline,
+#' and reference region.
+#' Also allows for custom formula parameters.
+#'
+#' @param tracer Amyloid PET tracer from ag_neuro dataset
+#'
+#'   A character string is expected.
+#'   When custom parameters are not provided, accepted values are:
```

Comment:
As I understand it, not all accepted values are actually valid, depending what the values of `pipeline` and `ref_region` are. So, I would suggest making a list of accepted combinations in the details section instead, and have left the same comment for the other two parameters.

```suggestion
#'  See Details section for accepted values in combination with `pipeline` and `ref_region`.
```